XML/Other/ReplaceableTestParameters.xml

```diff
@@ -725,4 +725,8 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>CPU_OFFLINE_VM_REBOOT</ReplaceThis>
 		<ReplaceWith>yes</ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<ReplaceThis>FAILSAFE_SH_URL</ReplaceThis>
```

Comment:
Will it be better idea to start with "DPDK_"? My concern here is for user friendliness.

Reply:
Thanks!
```
--- a/XML/Other/ReplaceableTestParameters.xml
+++ b/XML/Other/ReplaceableTestParameters.xml
@@ -726,7 +726,7 @@ Scenario 2 : You need to run all the performance tests quickly.
                <ReplaceWith>yes</ReplaceWith>
        </Parameter>
        <Parameter>
-               <ReplaceThis>FAILSAFE_SH_URL</ReplaceThis>
+               <ReplaceThis>DPDK_FAILSAFE_SH_URL</ReplaceThis>
```
```
--- a/XML/TestCases/FunctionalTests-DPDK.xml
+++ b/XML/TestCases/FunctionalTests-DPDK.xml
@@ -184,7 +184,7 @@
         <TestParameters>
             <param>dpdkSrcLink="DPDK_SOURCE_URL"</param>
             <param>pmds=mlx,vdev_netvsc,tap,failsafe,netvsc</param>
-            <param>failsafeUrl="FAILSAFE_SH_URL"</param>
+            <param>failsafeUrl="DPDK_FAILSAFE_SH_URL"</param>
```